osmose_config.py

```diff
@@ -492,7 +492,6 @@ class gen(default_country):
     'merge_school_FR',
     'merge_college_FR',
     'merge_service_public_FR',
-    'merge_pitch_FR',
```

Comment:
Why removing pitch ?

Reply:
As I said in the commit message, the data source doesn't exist anymore in CSV format, only XLSX. Can it be used ?

Reply:
Indeed, there is no longer the source. On the other hand, the analysis continues to run on its cache. It is still possible to keep it while waiting to mogration.

Reply:
Shouldn't the CSV data be committed in the [`merge_data`](https://github.com/osm-fr/osmose-backend/tree/master/merge_data) directory then ? If not, any user not having the cache (or future tests) will have errors on this Analyse. Or when redeploying osmose-backend servers, dropping caches etc.

I don't have access to osmose-backend server(s), can you send it by committing to this branch or one of yours ? Thank you

Reply:
I agree it is not a solution. But just keep it working as is.
I want to remove any data from `merge_data`.
Data from cache: https://github.com/osm-fr/osmose-backend/issues/1146#issuecomment-770222370
`analyser_merge_pitch_FR.py` can also be let untouched.

Reply:
Done